Code/Framework/AzCore/AzCore/Memory/BestFitExternalMapSchema.h

```diff
@@ -45,26 +45,27 @@ namespace AZ
             static const int        m_memoryBlockAlignment = 16;
             void*                   m_memoryBlock;              ///< Pointer to memory to allocate from. Can be uncached.
             unsigned int            m_memoryBlockByteSize;      ///< Sizes if the memory block.
-            IAllocatorAllocate*     m_mapAllocator;             ///< Allocator for the free chunks map. If null the SystemAllocator will be used.
+            IAllocator*             m_mapAllocator;             ///< Allocator for the free chunks map. If null the SystemAllocator will be used.
         };
 
         BestFitExternalMapSchema(const Descriptor& desc);
 
-        pointer_type    Allocate(size_type byteSize, size_type alignment, int flags);
-        void            DeAllocate(pointer_type ptr);
-        size_type       AllocationSize(pointer_type ptr);
+        pointer_type Allocate(size_type byteSize, size_type alignment, int flags = 0, const char* name = 0, const char* fileName = 0, int lineNum = 0, unsigned int suppressStackRecord = 0) override;
```

Comment:
The `const char*` default values should be nullptr
```suggestion
        pointer_type Allocate(size_type byteSize, size_type alignment, int flags = 0, const char* name = nullptr, const char* fileName = nullptr, int lineNum = 0, unsigned int suppressStackRecord = 0) override;
```